src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -741,6 +768,22 @@ describe('Step Review', () => {
             name: 'Packages'
         });
     });
+
+    test('has two tabs for centos', async () => {
+        setUpCentOS();
+
+        const buttonTarget = screen.getByTestId('tab-target');
+        const buttonSystem = screen.getByTestId('tab-system');
+
```

Comment:
Could you add a non-existence check here? A good way to do that is you could query by testid using the react testing library and then check that it is not in the document using jest-dom.